.github/workflows/create_release.yml

```diff
@@ -0,0 +1,37 @@
+# yamllint --format github .github/workflows/create_release.yml
+---
+name: create_release
+
+# We create a release version on a trigger tag, regardless of if the commit is documentation-only.
+#
+# See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+on:
+  push:
+    tags: 'release-[0-9]+.[0-9]+.[0-9]+**'  # Ex. release-1.2.3
```

Comment:
Didn't realize this isn't a regex in the last PR 😢

Reply:
show me a perfect proofreader and I'll show you a bridge for sale.. It's in Brooklyn!